### PR TITLE
Adding new support pages

### PR DESCRIPTION
**Pull request by mattwbeard**
New pages for options to contact Codat and to raise a new ticket with support

# Description

Adding new pages for raising tickets with support

image

## Type of change

Please delete options that are not relevant.

- [X] New document(s)/updating existing

## Reviews and merging

You are responsible for getting your PR merged. Address review comments promptly and **make sure to merge the PR when ready**.
Feel free to 'Enable automerge' - your PR will automatically merge when accepted and passing the build.

### CODE REVIEW

**Comment by mattwbeard** [comment]
@pzaichkina I think that's done it 🙏

File: src/pages/support/index.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                here
+              </a>{" "}
+              to sign up for your free plan, or{" "}
+              <a
+                href="https://www.codat.io/plans/#get-in-touch"
+                target="_blank"
+              >
+                contact our sales team
+              </a>{" "}
+              to schedule a demo and learn about Enterprise Pricing
```
Comment (pzaichkina) [inline src/pages/support/index.tsx:82]:
```suggestion
              to schedule a demo and learn about Enterprise pricing
```

**Comment by pzaichkina** [comment]
sorry, one more question @mattwbeard. I'm confused about where exactly in the docs this will be - please can you clarify how people will get to this page?

**Comment by mattwbeard** [comment]
They won't at the moment. They'll eventually go under a new menu that maybe combines some of the existing, but they're still a work in progress

**Review by pzaichkina — APPROVED** [review]

**Pull request merged**